### PR TITLE
Add: Fun category and approved command

### PR DESCRIPTION
**Pull request by SharifPoetra**

### CODE REVIEW

File: src/commands/fun/approved.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+			devOnly: false,
+			guildOnly: true,
+			nsfw: false, 
+			clientPermission: ["ATTACH_FILES"], 
+			authorPermission: []
+		});
+	}
+	execute(client, message, args, thread) {
+		const user = message.mentions.users.first() || client.users.get(args[0]) || message.author;
+		const { body } = thread.sync(get(`https://emilia-api.glitch.me/api/approved?image=${user.displayAvatarURL({ format: "png", size: 2048 })}`));
```
Comment (dwicao) [inline src/commands/fun/approved.js]: I think size `2048` is too large, can you decrease that to maybe `512`?

Comment (SharifPoetra) [reply src/commands/fun/approved.js]: but the resulting image from the api will be blurred

Comment (SharifPoetra) [reply src/commands/fun/approved.js]: I recommend use `2048` for a better quality, Are sure you want to decrease it?

Comment (dwicao) [reply src/commands/fun/approved.js]: how about `1024` ?

Comment (SharifPoetra) [reply src/commands/fun/approved.js]: Alright

**Comment by SharifPoetra** [comment]
So do you want to just looking or merge this now? :3

**Comment by dwicao** [comment]
if you're done, you can merge this, thanks

**Pull request merged**